cg/store/api/find.py

```diff
@@ -29,6 +29,10 @@ def customer_by_id(self, id_: int) -> models.Customer:
     def user(self, email: str) -> models.User:
         """Fetch a user from the store."""
         return self.User.query.filter_by(email=email).first()
+        
```

Comment:
please delete method if your not using it

Reply:
I didnt write this function so im not sure its not being used in some other code.

Reply:
The '+' signs in github indicates that the 'user_by_cust' got added in your commit, right? And if you don't call it, it is no longer needed, right?

Reply:
removed, great